cli/src/main/java/keywhiz/cli/commands/DescribeAction.java

```diff
@@ -89,7 +89,7 @@ public DescribeAction(DescribeActionConfig describeActionConfig, KeywhizClient c
           sanitizedSecret = keywhizClient.getSanitizedSecretByName(name);
 
           printing.printSanitizedSecretWithDetails(sanitizedSecret,
-              Arrays.asList("groups", "clients", "metadata"));
+              Arrays.asList("groups", "clients", "secret_data"));
```

Comment:
A quick inspection looks like we only ever set the same options ... Maybe we can just get rid of the Options thing?